rust/Cargo.toml

```diff
@@ -25,7 +25,7 @@ backoff = { version = "0.4", features = ["tokio"] }
 tracing = { version = "0.1.40" }
 tracing-subscriber = { version = "0.3.17", features = ["parking_lot"] }
 secrecy = "0.8"
-hickory-resolver = { version = "0.24", features = ["tokio-runtime"] }
+hickory-resolver = { git = "https://github.com/hickory-dns/hickory-dns", branch="main", features = ["tokio-runtime"] }
```

Comment:
let's se the `rev` to specific commit instead of `branch` to main, to have a consistent version across builds

Reply:
ah it's already on the `TODO` list